emily_landi/.gitignore

```diff
@@ -0,0 +1 @@
+**/node_modules
```

Comment:
Although you added node_modules to .gitignore, they were still committed (did you add the line to .gitignore after committing?). If you remove node_modules from your submission, I'll merge your pull request.